control/preimage/src/commands.rs

```diff
@@ -358,3 +360,336 @@ pub fn register_ether(params: &RegisterEtherArgs) -> (AssetHubRuntimeCall, Asset
+                    GeneralKey {
+                        length: 2,
+                        data: hex!(
+                            "0001000000000000000000000000000000000000000000000000000000000000"
+                        ),
+                    },
+                ]),
+            }),
+            "Bifrost Native Token",
+            "BNC",
+            12u8,
+        ),
+        register_polkadot_native_asset(
+            VersionedLocation::V4(Location {
+                parents: 1,
+                interior: X2([
+                    Parachain(2030),
+                    GeneralKey {
+                        length: 2,
+                        data: hex!(
+                            "0900000000000000000000000000000000000000000000000000000000000000"
+                        ),
+                    },
+                ]),
+            }),
+            "Voucher DOT",
+            "vDOT",
+            10u8,
+        ),
+        register_polkadot_native_asset(
+            VersionedLocation::V4(Location {
+                parents: 1,
+                interior: X2([Parachain(2034), GeneralIndex(0)]),
+            }),
+            "Hydration",
+            "HDX",
+            12u8,
+        ),
+        register_polkadot_native_asset(
+            VersionedLocation::V4(Location {
+                parents: 1,
+                interior: X1([Parachain(2039)]),
+            }),
+            "Integritee TEER",
+            "TEER",
+            12u8,
+        ),
+        register_polkadot_native_asset(
+            VersionedLocation::V4(Location {
+                parents: 1,
+                interior: X1([Parachain(2051)]),
+            }),
+            "Ajuna Polkadot AJUN",
+            "AJUN",
+            12u8,
+        ),
+        register_polkadot_native_asset(
+            VersionedLocation::V4(Location {
+                parents: 1,
+                interior: X1([Parachain(3344)]),
+            }),
+            "Polimec",
+            "PLMC",
+            10u8,
+        ),
+        register_polkadot_native_asset(
+            VersionedLocation::V4(Location {
+                parents: 1,
+                interior: X1([Parachain(3370)]),
+            }),
+            "LAOS",
+            "LAOS",
+            18u8,
+        ),
+        register_polkadot_native_asset(
+            VersionedLocation::V4(Location {
+                parents: 1,
+                interior: X1([Parachain(2086)]),
+            }),
+            "KILT Spiritnet",
+            "KILT",
+            15u8,
+        ),
+        register_polkadot_native_asset(
+            VersionedLocation::V4(Location {
+                parents: 1,
+                interior: X1([Parachain(2006)]),
+            }),
+            "Astar",
+            "ASTR",
+            18u8,
+        ),
+        register_polkadot_native_asset(
+            VersionedLocation::V4(Location {
+                parents: 1,
+                interior: X2([
+                    Parachain(2006),
```

Comment:
```suggestion
                    Parachain(2031),
```

Is Centrifuge's parachain ID not `2031`?

![Screenshot 2025-03-19 at 09 33 45](https://github.com/user-attachments/assets/23482175-d7f1-49f2-9f03-44363148dc5b)

Reply:
The referenda is already up. You can cancel it and repost? Or you can let it go through and register Cetnrifuge in the next batch. The only downside is that there will be an asset with the incorrect metadata squatting that multi-locaiton.